src/widgets/resourceList.js

```diff
@@ -0,0 +1,519 @@
+import { Op, QueryTypes } from 'sequelize';
```

Comment:
Seems like this file might be fairly temporary, but some thoughts about this file

- some comments as we move through this, so that future engineers or myself don't get confused and give up if we have to change this. Even the jsdoc comments would be a great start, but I'd also like to know what is happening without having to follow the path of the code

- it might be nice to break this into multiple files just for readability